### ISSUE TITLE
Weather icons are not correct

### ISSUE THREAD

**Issue by asjmcguire** [home-assistant/frontend#14643]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The weather icons should be showing snow - but they are showing what looks like light rain.

Here is the more info dialog:

![image](https://user-images.githubusercontent.com/45770738/206461537-2cd2aca5-ac38-45f0-9e36-de13b1bdc114.png)

which as you can see shows snow everyday.

Here is the actual weather card:

![image](https://user-images.githubusercontent.com/45770738/206461669-ee1fd3d7-cc9d-442e-bad9-8038c48c5684.png)

Which is not showing snow for any day.

I have checked and the OpenWeatherMap forecast is the same - the more info box shows snow on the forecasts, but the card itself does not.

Additionally I noticed last night that when it was partly cloudy, it was showing the partly cloudy with sun icon for the current conditions, not the partly cloudy with moon icon.

### Describe the behavior you expected

I expected that it would show snow instead of what looks like light rain.
Because this is the first time we had had snow forecast for a long time, I don't know what the last working version of this was.

### Steps to reproduce the issue

1. Add a weather entity
2. Update the forecast with snow (copy and paste from above)
3. Observe the icon displayed

### What version of Home Assistant Core has the issue?

core-2022.12.0

### In which browser are you experiencing the issue with?

Google Chrome Version 108.0.5359.98 (Official Build) (64-bit)

### Which operating system are you using to run this browser?

Windows 11

### State of relevant entities

```txt
temperature: 2.1
temperature_unit: °C
humidity: 84
pressure: 1009.25
pressure_unit: hPa
wind_bearing: 156
wind_speed: 0.89
wind_speed_unit: mph
visibility: 4.29
visibility_unit: mi
precipitation_unit: mm
forecast:
  - datetime: '2022-12-08T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 2
    templow: -2
    precipitation: 5
    precipitation_probability: 60
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 9.63
    wind_bearing: 314
    wind_direction_cardinal: NW
  - datetime: '2022-12-09T00:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: 2
    templow: -2
    precipitation: 1
    precipitation_probability: 30
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 8.64
    wind_bearing: 295
    wind_direction_cardinal: WNW
  - datetime: '2022-12-10T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 1
    templow: -3
    precipitation: 36.74
    precipitation_probability: 80
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 6.65
    wind_bearing: 149
    wind_direction_cardinal: SSE
  - datetime: '2022-12-11T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 1
    templow: -2
    precipitation: 9
    precipitation_probability: 70
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 5.47
    wind_bearing: 322
    wind_direction_cardinal: NW
  - datetime: '2022-12-12T00:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: 1
    templow: -3
    precipitation: 0
    precipitation_probability: 30
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 3.42
    wind_bearing: 182
    wind_direction_cardinal: S
  - datetime: '2022-12-13T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 0
    templow: -4
    precipitation: 6
    precipitation_probability: 60
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 3.17
    wind_bearing: 272
    wind_direction_cardinal: W
  - datetime: '2022-12-14T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 1
    templow: -3
    precipitation: 24
    precipitation_probability: 70
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 5.78
    wind_bearing: 342
    wind_direction_cardinal: NNW
  - datetime: '2022-12-15T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: -1
    templow: -4
    precipitation: 25
    precipitation_probability: 70
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 5.59
    wind_bearing: 340
    wind_direction_cardinal: NNW
  - datetime: '2022-12-16T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: -1
    templow: -5
    precipitation: 24
    precipitation_probability: 60
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 5.84
    wind_bearing: 322
    wind_direction_cardinal: NW
  - datetime: '2022-12-17T00:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: -1
    templow: -5
    precipitation: 24
    precipitation_probability: 40
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 5.34
    wind_bearing: 296
    wind_direction_cardinal: WNW
attribution: Powered by Home Assistant
friendly_name: WeatherFlow
```
That's the complete attributes for the entity.

This is the weather entity code:

```
  - platform: template
    name: WeatherFlow
    condition_template: "{{ states('sensor.hub_hb_00071788_weather') }}"
    temperature_template: "{{ states('sensor.tempest_st_00015638_temperature') | float(0)}}"
    humidity_template: "{{ states('sensor.tempest_st_00015638_humidity')| int(0) }}"
    pressure_template: "{{ states('sensor.tempest_st_00015638_sea_level_pressure')| float(0) }}"
    wind_speed_template: "{{ ( states('sensor.tempest_st_00015638_wind_speed_avg') | float(1) * 18 / 5 ) | round(2) }}"
    wind_bearing_template: "{{ states('sensor.tempest_st_00015638_wind_bearing_avg')| int(0) }}"
    visibility_template: "{{ states('sensor.tempest_st_00015638_visibility')| float(0) }}"
    forecast_template: "{{ state_attr('sensor.hub_hb_00071788_weather', 'daily_forecast') }}"
    unique_id: wx-6ba7245a-24c6-402a-a823-3195e338bc4a
```

and this is the entity the forecast_template comes from:

```
attribution: Powered by WeatherFlow2MQTT
temp_high_today: 2
temp_low_today: -2
daily_forecast:
  - datetime: '2022-12-08T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 2
    templow: -2
    precipitation: 5
    precipitation_probability: 60
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 15.5
    wind_bearing: 314
    wind_direction_cardinal: NW
  - datetime: '2022-12-09T00:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: 2
    templow: -2
    precipitation: 1
    precipitation_probability: 30
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 13.9
    wind_bearing: 295
    wind_direction_cardinal: WNW
  - datetime: '2022-12-10T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 1
    templow: -3
    precipitation: 36.74
    precipitation_probability: 80
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.7
    wind_bearing: 149
    wind_direction_cardinal: SSE
  - datetime: '2022-12-11T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 1
    templow: -2
    precipitation: 9
    precipitation_probability: 70
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 8.8
    wind_bearing: 322
    wind_direction_cardinal: NW
  - datetime: '2022-12-12T00:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: 1
    templow: -3
    precipitation: 0
    precipitation_probability: 30
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 5.5
    wind_bearing: 182
    wind_direction_cardinal: S
  - datetime: '2022-12-13T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 0
    templow: -4
    precipitation: 6
    precipitation_probability: 60
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 5.1
    wind_bearing: 272
    wind_direction_cardinal: W
  - datetime: '2022-12-14T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 1
    templow: -3
    precipitation: 24
    precipitation_probability: 70
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 9.3
    wind_bearing: 342
    wind_direction_cardinal: NNW
  - datetime: '2022-12-15T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: -1
    templow: -4
    precipitation: 25
    precipitation_probability: 70
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 9
    wind_bearing: 340
    wind_direction_cardinal: NNW
  - datetime: '2022-12-16T00:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: -1
    templow: -5
    precipitation: 24
    precipitation_probability: 60
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 9.4
    wind_bearing: 322
    wind_direction_cardinal: NW
  - datetime: '2022-12-17T00:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: -1
    templow: -5
    precipitation: 24
    precipitation_probability: 40
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 8.6
    wind_bearing: 296
    wind_direction_cardinal: WNW
hourly_forecast:
  - datetime: '2022-12-08T14:00:00+00:00'
    conditions: Cloudy
    condition: cloudy
    temperature: 2
    pressure: 1010.86
    humidity: 84
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-sleet
    precip_type: sleet
    wind_speed: 18
    wind_gust: 28.8
    wind_bearing: 330
    wind_direction_cardinal: NNW
    uv: 0
    feels_like: -3
  - datetime: '2022-12-08T15:00:00+00:00'
    conditions: Wintry Mix Likely
    condition: snowy-rainy
    temperature: 2
    pressure: 1010.66
    humidity: 84
    precipitation: 0
    precipitation_probability: 50
    precip_icon: chance-sleet
    precip_type: sleet
    wind_speed: 18
    wind_gust: 28.8
    wind_bearing: 330
    wind_direction_cardinal: NNW
    uv: 0
    feels_like: -3
  - datetime: '2022-12-08T16:00:00+00:00'
    conditions: Wintry Mix Possible
    condition: snowy-rainy
    temperature: 1
    pressure: 1010.67
    humidity: 85
    precipitation: 0
    precipitation_probability: 35
    precip_icon: chance-sleet
    precip_type: sleet
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 329
    wind_direction_cardinal: NNW
    uv: 0
    feels_like: -3
  - datetime: '2022-12-08T17:00:00+00:00'
    conditions: Snow Likely
    condition: snowy
    temperature: 1
    pressure: 1010.69
    humidity: 84
    precipitation: 4
    precipitation_probability: 60
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 18
    wind_gust: 28.8
    wind_bearing: 326
    wind_direction_cardinal: NW
    uv: 0
    feels_like: -4
  - datetime: '2022-12-08T18:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: 0
    pressure: 1010.66
    humidity: 85
    precipitation: 1
    precipitation_probability: 30
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 28.8
    wind_bearing: 321
    wind_direction_cardinal: NW
    uv: 0
    feels_like: -5
  - datetime: '2022-12-08T19:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -1
    pressure: 1010.47
    humidity: 87
    precipitation: 0
    precipitation_probability: 5
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 25.2
    wind_bearing: 312
    wind_direction_cardinal: NW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-08T20:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -1
    pressure: 1010.36
    humidity: 86
    precipitation: 0
    precipitation_probability: 5
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 305
    wind_direction_cardinal: NW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-08T21:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -2
    pressure: 1010.16
    humidity: 84
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 302
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-08T22:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -2
    pressure: 1009.82
    humidity: 81
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 296
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-08T23:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -2
    pressure: 1009.39
    humidity: 80
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 28.8
    wind_bearing: 293
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -7
  - datetime: '2022-12-09T00:00:00+00:00'
    conditions: Clear
    condition: clear-night
    temperature: -2
    pressure: 1008.97
    humidity: 79
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 28.8
    wind_bearing: 284
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -7
  - datetime: '2022-12-09T01:00:00+00:00'
    conditions: Clear
    condition: clear-night
    temperature: -2
    pressure: 1008.37
    humidity: 78
    precipitation: 0
    precipitation_probability: 5
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 18
    wind_gust: 28.8
    wind_bearing: 281
    wind_direction_cardinal: W
    uv: 0
    feels_like: -7
  - datetime: '2022-12-09T02:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: -2
    pressure: 1007.87
    humidity: 75
    precipitation: 0
    precipitation_probability: 20
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 18
    wind_gust: 28.8
    wind_bearing: 284
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -7
  - datetime: '2022-12-09T03:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: -1
    pressure: 1007.42
    humidity: 77
    precipitation: 0
    precipitation_probability: 20
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 21.6
    wind_gust: 36
    wind_bearing: 290
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -7
  - datetime: '2022-12-09T04:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: -1
    pressure: 1007.09
    humidity: 78
    precipitation: 1
    precipitation_probability: 30
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 18
    wind_gust: 32.4
    wind_bearing: 304
    wind_direction_cardinal: NW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-09T05:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -1
    pressure: 1006.62
    humidity: 79
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 18
    wind_gust: 32.4
    wind_bearing: 300
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-09T06:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -1
    pressure: 1006.47
    humidity: 80
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 18
    wind_gust: 32.4
    wind_bearing: 300
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-09T07:00:00+00:00'
    conditions: Clear
    condition: clear-night
    temperature: -1
    pressure: 1006.29
    humidity: 83
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 28.8
    wind_bearing: 300
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-09T08:00:00+00:00'
    conditions: Clear
    condition: sunny
    temperature: -1
    pressure: 1006.32
    humidity: 82
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 300
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-09T09:00:00+00:00'
    conditions: Clear
    condition: sunny
    temperature: -1
    pressure: 1006.29
    humidity: 83
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 297
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-09T10:00:00+00:00'
    conditions: Clear
    condition: sunny
    temperature: -1
    pressure: 1006.17
    humidity: 81
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 289
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -4
  - datetime: '2022-12-09T11:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: 1
    pressure: 1005.97
    humidity: 76
    precipitation: 0
    precipitation_probability: 5
    precip_icon: chance-sleet
    precip_type: sleet
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 291
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -3
  - datetime: '2022-12-09T12:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: 2
    pressure: 1005.69
    humidity: 75
    precipitation: 0
    precipitation_probability: 5
    precip_icon: chance-rain
    precip_type: rain
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 295
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -2
  - datetime: '2022-12-09T13:00:00+00:00'
    conditions: Clear
    condition: sunny
    temperature: 2
    pressure: 1005.42
    humidity: 76
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-rain
    precip_type: rain
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 289
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -2
  - datetime: '2022-12-09T14:00:00+00:00'
    conditions: Clear
    condition: sunny
    temperature: 2
    pressure: 1005.27
    humidity: 78
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-rain
    precip_type: rain
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 289
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -2
  - datetime: '2022-12-09T15:00:00+00:00'
    conditions: Clear
    condition: sunny
    temperature: 0
    pressure: 1005.22
    humidity: 81
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-sleet
    precip_type: sleet
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 290
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -4
  - datetime: '2022-12-09T16:00:00+00:00'
    conditions: Clear
    condition: clear-night
    temperature: -1
    pressure: 1005.17
    humidity: 85
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 14.4
    wind_gust: 18
    wind_bearing: 291
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -5
  - datetime: '2022-12-09T17:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: -1
    pressure: 1005.12
    humidity: 86
    precipitation: 0
    precipitation_probability: 25
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 296
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -5
  - datetime: '2022-12-09T18:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: -1
    pressure: 1005.17
    humidity: 87
    precipitation: 0
    precipitation_probability: 25
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 301
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -5
  - datetime: '2022-12-09T19:00:00+00:00'
    conditions: Snow Possible
    condition: snowy
    temperature: -2
    pressure: 1005.27
    humidity: 88
    precipitation: 0
    precipitation_probability: 25
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 302
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -5
  - datetime: '2022-12-09T20:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -2
    pressure: 1005.37
    humidity: 88
    precipitation: 0
    precipitation_probability: 15
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 304
    wind_direction_cardinal: NW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-09T21:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -2
    pressure: 1005.47
    humidity: 88
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 304
    wind_direction_cardinal: NW
    uv: 0
    feels_like: -5
  - datetime: '2022-12-09T22:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -2
    pressure: 1005.52
    humidity: 91
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 305
    wind_direction_cardinal: NW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-09T23:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -2
    pressure: 1005.57
    humidity: 90
    precipitation: 0
    precipitation_probability: 10
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 303
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-10T00:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -2
    pressure: 1005.57
    humidity: 91
    precipitation: 0
    precipitation_probability: 15
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 291
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -6
  - datetime: '2022-12-10T01:00:00+00:00'
    conditions: Partly Cloudy
    condition: partlycloudy
    temperature: -3
    pressure: 1005.57
    humidity: 90
    precipitation: 0
    precipitation_probability: 15
    precip_icon: chance-snow
    precip_type: snow
    wind_speed: 10.8
    wind_gust: 10.8
    wind_bearing: 290
    wind_direction_cardinal: WNW
    uv: 0
    feels_like: -7
icon: mdi:chart-box-outline
friendly_name: Hub HB-00071788 Weather
```
```


### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

**Comment by spacegaier** [comment]
Looks like we are missing a matching SVG path definition for snow currently.

**Comment by asjmcguire** [comment]
I had a look in weather.ts and can see the SVG for snowy, so I pasted it in to an online viewer. I then compared the image to what it shown in the mobile app where I can zoom in a bit to see it.

The SVG is 3 circles, 2 larger and a smaller one between them. That is in fact what is being shown. The problem is that because the colour (blue) of the snow is the same as the colour for rain - it's not obvious that it's not rain.
